common.blocks/select/select.bemhtml

```diff
@@ -21,7 +21,7 @@ block('select')(
                         iterateOptions(option.group);
                     } else {
                         firstOption || (firstOption = option);
-                        optionIds.push(option.id = _this.generateId());
+                        optionIds.push(option.id = 'option' + random++);
```

Comment:
instead of `'option' + random++` (with `random` variable) we can just use `_this.generateId() + 'option' + i`